src/panels/lovelace/special-rows/hui-section-row.ts

```diff
@@ -41,14 +41,14 @@ class HuiSectionRow extends LitElement implements LovelaceRow {
   static get styles(): CSSResult {
     return css`
       .label {
-        color: var(--primary-color);
+        color: var(--secondary-text-color);
```

Comment:
Why would `primary-color` give problems with dark themes? You will have multiple other problems in that case, like links that are not visible etc.

What is your `primary-color`?

Reply:
I don't know off hand; it's something I've noticed in some dark themes.  Just seems to me like the UI should be more consistently using text colors for text and divider colors for dividers, since otherwise a lot of weirdness and unexpected side-effects happen when trying to build a very dark theme

Reply:
We have an official dark theme now, and that seems to work fine.